### PR TITLE
Updates the documentation for manage_the_buzz_for_developers

### PR DESCRIPTION
**Pull request by sohinihm4**
Below items still need to be added in the documentation:
1. Flow diagrams for c2c/spa/native apps
2. More steps in quick start guide for native apps
3. Update service app flow diagram
4. I have used existing folder structure, But we may have to change it a bit if required as there are few extra pages generated

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+venv/
```
Comment (manavkumarm) [inline .gitignore:1]: Please remove all the sphinx files. They are already being added in PR # 1 (from Akshay). If you have anything to add beyond what Akshay has added, let's discuss and then you can submit it as a separate PR.

File: source/index.rst [hunk 2]
```diff
@@ -1,7 +1,10 @@
 Welcome to the COCO Documentation
 =================================
 
-Learn all about COCO here. We introduce you to the concepts and architecture, allow you to get started quickly with the various platform API's with explained code examples, and finally allow you to deep dive into the API Documentation and build your own COCO-based solution.
+Learn all about COCO here. We introduce you to the concepts and architecture,
```
Comment (manavkumarm) [inline source/index.rst]: I don't see any change in the content. Please revert this file to my version. The line breaks here don't make any difference to how the file appears in HTML or the RST preview.

File: source/manage_the_buzz_for_developers/applications_and_user_system_manage_the_buzz.rst [hunk 3]
```diff
@@ -3,12 +3,9 @@
 Applications and User System
 ============================
 
-<Introduction to this sub-section - Owner: Manav>
```
Comment (manavkumarm) [inline source/manage_the_buzz_for_developers/applications_and_user_system_manage_the_buzz.rst]: Please don't delete these placeholders. They indicate content that is pending to be written.

File: source/manage_the_buzz_for_developers/applications_and_user_system_manage_the_buzz.rst [hunk 4]
```diff
@@ -3,12 +3,9 @@
...
-
 .. toctree::
    :maxdepth: 5
-   :caption: In This Section:
 
-   overview_applications_user_system_manage_the_buzz
-   custom_solution_for_self_managed_user_system
-   value_added_service_extensions_for_coco_solutions
+   applications_and_user_system_manage_the_buzz/overview_applications_user_system_manage_the_buzz
```
Comment (manavkumarm) [inline source/manage_the_buzz_for_developers/applications_and_user_system_manage_the_buzz.rst:11]: Revert all changes similar to this. They are already done in PR # 1.

File: source/manage_the_buzz_for_developers/applications_and_user_system_manage_the_buzz/custom_solution_for_self_managed_user_system/quick_start_guide_service_apps.rst [hunk 5]
```diff
@@ -1,6 +1,34 @@
 .. _quick_start_guide_service_apps:
 
-Quick Start Guide - Service Apps
-================================
+Quick Start Guide
```
Comment (manavkumarm) [inline source/manage_the_buzz_for_developers/applications_and_user_system_manage_the_buzz/custom_solution_for_self_managed_user_system/quick_start_guide_service_apps.rst]: Revert the title to what I had. I believe it makes the context of this clear, because there is no "Service Apps" menu option that comes before this. For the other quick start guides, the prior menu option provides the necessary context and therefore I kept them as just "Quick Start Guide'.

**Review by manavkumarm — CHANGES_REQUESTED** [review]

**Comment by sohinihm4** [comment]
Closing this PR as another is already created for same.
